RecoMET/METFilters/python/ecalBadCalibFilter_cfi.py

```diff
@@ -30,3 +30,20 @@
                                                        872420273,872436907,872420147,872439731,
                                                        872436657,872420397,872439732,872439339,
                                                        872439603])
+
+from Configuration.Eras.Modifier_run2_miniAOD_devel_cff import run2_miniAOD_devel
+from Configuration.Eras.Modifier_run2_nanoAOD_94XMiniAODv1_cff import run2_nanoAOD_94XMiniAODv1
+from Configuration.Eras.Modifier_run2_nanoAOD_94XMiniAODv2_cff import run2_nanoAOD_94XMiniAODv2
+from Configuration.Eras.Modifier_run2_nanoAOD_102Xv1_cff import run2_nanoAOD_102Xv1
+for modifier in run2_miniAOD_devel, run2_nanoAOD_94XMiniAODv1, run2_nanoAOD_94XMiniAODv2, run2_nanoAOD_102Xv1:
+    modifier.toModify(ecalBadCalibFilter, baddetEcal = [872439604,872422825,872420274,872423218,
+                                                        872423215,872416066,872435036,872439336,
+                                                        872420273,872436907,872420147,872439731,
+                                                        872436657,872420397,872439732,872439339,
+                                                        872439603,872422436,872439861,872437051,
+                                                        872437052,872420649,872421950,872437185,
```

Comment:
please do not copy-paste already existing IDs from ```run2_common```.
This part here should contain just the additions.
Consider also that the common nanoAOD overrides may better be grouped into one modifier.


Also, a recommended syntax for multiple modifiers is using ```(a | b).toModify```
So, ```(run2_miniAOD_devel | run2_nanoAOD_94XMiniAODv1 |  run2_nanoAOD_94XMiniAODv2 | run2_nanoAOD_102Xv1).toModify

Reply:
How can I extend the list?

Reply:
```+= [a, b, c]``` ?